docs/template_plugin/backend/int_executable.cpp

```diff
@@ -154,8 +154,7 @@ bool runtime::interpreter::INTExecutable::call(const vector<shared_ptr<runtime::
             if (!variable_context.get_variable_value(variable)) {
                 auto h_tensor = std::make_shared<ngraph::HostTensor>(cloned_node->get_input_element_type(0),
                                                                      cloned_node->get_input_shape(0));
-                std::vector<float> data(ov::shape_size(cloned_node->get_input_shape(0)), 0);
-                h_tensor->write(data.data(), data.size() * sizeof(float));
+                h_tensor->write(h_tensor->get_data_ptr(), h_tensor->get_size_in_bytes());
```

Comment:
Why do we need this change?

Reply:
With `float`, 
- Test cases are passed only with `f32`, `i32`, `u32`. 
- With other ngraph element type in test cases, it produces mismatch error in `runtime::HostTensor::write()` in host_tensor.cpp between `n` and `m_buffer_size`. 

FYI, I checked with below types
- `double` : then only  `f64`, `i64`, `u64 `are passed.
- `float` : then only `f32`, `i32`, `u32 `are passed.
- `float16` : then only `f16`, `i16`, `u16 `are passed.
- `int8_t` : then only `i8`, `u8 `are passed.

Thus, the method needs to be generalized, that the element type should be corresponding to that of `h_tensor` which is created from `cloned_node`.